pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -741,14 +751,42 @@ monitorLoop:
 			logger.Reason(err).Error("failed to get domain job info")
 			break
 		}
+		log.Log.Infof("Debug2: stats-type	%d", stats.Type)
+		log.Log.Infof("%d, %d, %d, %d, %d", libvirt.DOMAIN_JOB_UNBOUNDED, libvirt.DOMAIN_JOB_NONE, libvirt.DOMAIN_JOB_COMPLETED, libvirt.DOMAIN_JOB_CANCELLED, libvirt.DOMAIN_JOB_FAILED)
+
+		jobStatsInfo, err := dom.GetJobStats(0)
+		if err != nil {
+			logger.Reason(err).Error("failed to get domain stats job info")
+			break
+		}
+		log.Log.Infof("Debug Job Stats: %d", jobStatsInfo.DataRemaining)
+		log.Log.Infof("Debug Job Stats: %d", jobStatsInfo.DataProcessed)
+		log.Log.Infof("Debug Job Stats: %d", jobStatsInfo.MemDirtyRate)
+		log.Log.Infof("Debug Job Stats: %d", jobStatsInfo.Type)
 
 		remainingData := int64(stats.DataRemaining)
+		log.Log.Infof("Debug2 Job Info: stats-dataremaining %d", stats.DataRemaining)
+		log.Log.Infof("Debug2 Job Info: stats-dataprocessed %d", stats.DataProcessed)
+		log.Log.Infof("Debug2 Job Info: stats-memorydirtyrate %d", stats.MemDirtyRate)
+		log.Log.Infof("Debug2 Job Info: stats-type %d", stats.Type)
+
 		switch stats.Type {
 		case libvirt.DOMAIN_JOB_UNBOUNDED:
 			// Migration is running
 			now := time.Now().UTC().Unix()
 			elapsed := now - start
 
+			/*			l.migrateInfoStats.DataProcessed = jobStatsInfo.DataProcessed
+						l.migrateInfoStats.DataProcessedSet = jobStatsInfo.DataProcessedSet
+						l.migrateInfoStats.DataRemaining = jobStatsInfo.DataRemaining
+						l.migrateInfoStats.DataRemainingSet = jobStatsInfo.DataRemainingSet
+						l.migrateInfoStats.MemDirtyRate = jobStatsInfo.MemDirtyRate
+						l.migrateInfoStats.MemDirtyRateSet = jobStatsInfo.MemDirtyRateSet*/
+			l.migrateInfoStats = statsconv.Convert_libvirt_DomainJobInfo_To_stats_DomainJobInfo(jobStatsInfo)
```

Comment:
you could then use `stats` instead of jobStatsInfo

Reply:
Continuing to use `jobStatsinfo` as `stats` is also a package imported into the file.